dashboard/app/views/levels/_form.html.haml

```diff
@@ -7,7 +7,7 @@
       = f.label :name
       = f.text_field :name, placeholder: 'Level Name'
   / Only enter name to create new records of certain level types.
-  - unless @level.new_record? && (@level.is_a?(Grid) || @level.is_a?(Calc) || @level.is_a?(Eval) || @level.is_a?(Artist) || @level.is_a?(Applab))
+  - unless @level.new_record? && (@level.is_a?(Grid) || @level.is_a?(Calc) || @level.is_a?(Eval) || @level.is_a?(Artist) || @level.is_a?(Applab) || @level.is_a?(NetSim))
```

Comment:
This seems like a great candidate for extracting into a property of Game or of the class, but I don't trust my intuition on dashboard stuff yet.  Thoughts?